wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -158,6 +158,44 @@ class SwiftGenerator private constructor(
       else -> null
     }
 
+  private val Field.defaultedValue: CodeBlock?
+    get() {
+      when (val default = default) {
+        // identity values
+        null -> {
+          if (!isOptional) return null
+          if (type == ProtoType.ANY) return null
+          if (isMessage && !isRequiredParameter && !isCollection) {
+            val messageType = schema.getType(type!!) as MessageType
+            return if (messageType.fields.any { it.isRequiredParameter }) null else CodeBlock.of("%T()", messageType.typeName.makeNonOptional())
+          }
+          if (isEnum) {
+            val enumType = schema.getType(type!!) as EnumType
+            return if (enumType.constants.getOrNull(0) != null) CodeBlock.of("%T.%L", typeName.makeNonOptional(), enumType.constants[0].name) else null
```

Comment:
Is `enumType.constants.getOrNull(0)` correct? That just looks like the first (mapped) constant, _not_ the 0-th value.

Reply:
Based on the proto3 code, I think we want `enum.constants.filter { it.tag == 0 }.firstOrNull()`

Reply:
Yes this is probably incorrect. There is also `enum.constant(0)` but I think this didnt work, will be updating and looking again.

Reply:
fixed